CRM/Contact/Page/AJAX.php

```diff
@@ -786,29 +786,29 @@ public static function getDedupes() {
 
     $count = 0;
     foreach ($dupePairs as $key => $pairInfo) {
-      $pair =& $pairInfo['data'];
+      $pair = $pairInfo['data'];
```

Comment:
The lines are all too long anyway, and we might save a bit of memory/processing time? :3